Adding dependency `anaconda/2020.11:python3` to your environment
Adding package `my_py_env/20221129` to your environment
/home/2688/.conda/my_py_env/20221129/lib/python3.10/site-packages/MDAnalysis/topology/guessers.py:146: UserWarning: Failed to guess the mass for the following atom types: H11
  warnings.warn("Failed to guess the mass for the following atom types: {}".format(atom_type))
/home/2688/.conda/my_py_env/20221129/lib/python3.10/site-packages/MDAnalysis/topology/guessers.py:146: UserWarning: Failed to guess the mass for the following atom types: H12
  warnings.warn("Failed to guess the mass for the following atom types: {}".format(atom_type))
/home/2688/.conda/my_py_env/20221129/lib/python3.10/site-packages/MDAnalysis/topology/guessers.py:146: UserWarning: Failed to guess the mass for the following atom types: H21
  warnings.warn("Failed to guess the mass for the following atom types: {}".format(atom_type))
/home/2688/.conda/my_py_env/20221129/lib/python3.10/site-packages/MDAnalysis/topology/guessers.py:146: UserWarning: Failed to guess the mass for the following atom types: H22
  warnings.warn("Failed to guess the mass for the following atom types: {}".format(atom_type))
/home/2688/.conda/my_py_env/20221129/lib/python3.10/site-packages/MDAnalysis/topology/guessers.py:146: UserWarning: Failed to guess the mass for the following atom types: H31
  warnings.warn("Failed to guess the mass for the following atom types: {}".format(atom_type))
/home/2688/.conda/my_py_env/20221129/lib/python3.10/site-packages/MDAnalysis/topology/guessers.py:146: UserWarning: Failed to guess the mass for the following atom types: H32
  warnings.warn("Failed to guess the mass for the following atom types: {}".format(atom_type))
/home/2688/.conda/my_py_env/20221129/lib/python3.10/site-packages/MDAnalysis/topology/guessers.py:146: UserWarning: Failed to guess the mass for the following atom types: N1
  warnings.warn("Failed to guess the mass for the following atom types: {}".format(atom_type))
/home/2688/.conda/my_py_env/20221129/lib/python3.10/site-packages/MDAnalysis/topology/guessers.py:146: UserWarning: Failed to guess the mass for the following atom types: N2
  warnings.warn("Failed to guess the mass for the following atom types: {}".format(atom_type))
/home/2688/.conda/my_py_env/20221129/lib/python3.10/site-packages/MDAnalysis/topology/guessers.py:146: UserWarning: Failed to guess the mass for the following atom types: N3
  warnings.warn("Failed to guess the mass for the following atom types: {}".format(atom_type))
/home/2688/.conda/my_py_env/20221129/lib/python3.10/site-packages/MDAnalysis/topology/guessers.py:146: UserWarning: Failed to guess the mass for the following atom types: HW1
  warnings.warn("Failed to guess the mass for the following atom types: {}".format(atom_type))
/home/2688/.conda/my_py_env/20221129/lib/python3.10/site-packages/MDAnalysis/topology/guessers.py:146: UserWarning: Failed to guess the mass for the following atom types: HW2
  warnings.warn("Failed to guess the mass for the following atom types: {}".format(atom_type))
/home/2688/.conda/my_py_env/20221129/lib/python3.10/site-packages/MDAnalysis/topology/guessers.py:146: UserWarning: Failed to guess the mass for the following atom types: OW
  warnings.warn("Failed to guess the mass for the following atom types: {}".format(atom_type))
